### DISCUSSION TITLE
Can't make normal OpenMPTCP, always has problems (0.993)

### DISCUSSION THREAD

**Discussion by zionkv** [Ysurac/openmptcprouter#1949] [Q&A]
Hello, I am using VirtualBOX image from openmptcp repo with passing through USB-modems as router and already tried a lot of VDS services like Selectel, FirstVDS with different configurations. I already tried all OS versions like Debian 9, 10, Ubuntu 20 as a Server. 
With Debian 10 script v.0.993 working incorrect, because not all components installed when I tried couple times.

I always get some problems. More often it's "Shadowsocks can't get public IP". If I will turn on vr2 - "vr2 can't get public IP".
Tried to re-install pi-hole.
Tried to turn off all firewalls.
Tried to Google "Server mptcp cant get external address", "Shadowsocks cant get IP", etc.
Maybe it's [0.993] critical problem? 

**How to make just normal installation with any OS, any VDS, any router?**

_*btw there is a mistake about port in description. The true SSH port is "65222" after install openmptcp, but its "65101" everywhere in descriptions and manuals._

**Comment by Ysurac** [comment]
I don't see where 65101 is used for SSH port in doc.
Script 0.993 is not supported since a very long time. Latest VPS script is 0.1026.

**Reply by zionkv** [reply]
Wow, you answered so fast!
I was read russian translation in habrahabr.ru and they use 65101.
After that I was installed 0.993 many times and in main screen system said that I should use 65101.

I am just using your instruction with ssh command for install openmptcpserver: 
Ouch, my mistake, its really  "OpenMPTCProuter VPS 0.1026". But still have same problems for my system and still have 65101 port.
I attached screenshot
![image](https://user-images.githubusercontent.com/3386407/125281776-379bb080-e349-11eb-8b98-c13549ffc7eb.png)
*installed 10 mins ago

Btw can I use OMR dhcp-client with my 4G USB-modems (NDIS mode)?

I think the better way to make it is make with one of your approved VDS like AWS\Vultr and make screenshots after full clean installation.

**Reply by Ysurac** [reply]
I think it's an error in this external doc.
I don't see error in the screenshot. Shadowsocks really use 65101.
Approved VPS, are only VPS that are tested by me or some user. This should work on most VPS that allow custom kernel and running under Debian 10.
No issue to use 4G USB modem, use usb0 (or the created interface name, check State->Kernel log or system log to check what is created)
